modules/alloy/docs/behaviours/Tooltipping.adoc

```diff
@@ -0,0 +1,82 @@
+== General Overview
```

Comment:
These are just rough notes. They might not make the final cut. We need to decide as a team if we are OK with having `adoc` files in our code repos.